pandas/core/groupby/generic.py

```diff
@@ -2150,6 +2151,7 @@ def value_counts(
     def fillna(
         self,
         value: Hashable | Mapping | Series | DataFrame = None,
+        *,
```

Comment:
This should raise a warning, not change the behavior

Reply:
Sorry, I requested review and forgot to push the code.

Have undo this change in behavior and now we simply raise FutureWarning